templates/themes/horizon/invenio_communities/details/home/index.html

```diff
@@ -119,19 +119,18 @@ <h1 class="ui large header">{{ _("Recent uploads") }}</h1>
       </div>
     {% endif %}
 
-    <div class="ui stackable theme-font grid container rel-mt-2">
+    <div class="ui stackable theme-font one column grid container rel-mt-2">
       <div class="column">
           <div class="row item">
             <h1 class="ui large header">{{ _("How it works") }}</h1>
           </div>
-          <div class="ui divider  ml-0"></div>
-          <div class="sixteen wide stretched column rel-mt-4">
-            <div class="ui equal width grid">
-              <div class="five wide computer sixteen wide mobile  segment-container column rel-m-1">
-                <section>
+          <div class="ui divider ml-0"></div>
+          <div class="ui wide stretched stackable three column grid rel-mt-3">
+            <div class="column">
+              <div class="segment-container rel-p-1">
                   <h1 class="center aligned ui medium header rel-m-1">{{ _("Submit your research") }} </h1>
-                  <div class="ui  bottom attached">
-                    <ul>
+                  <div class="ui bottom attached">
+                    <ul class="rel-pr-2">
```

Comment:
I added a little right padding to each list, since the bullet points add quite a bit of left margin and made it look uneven.